src/pages/create/recruitCreatePage/RecruitCreatePage.tsx

```diff
@@ -35,28 +33,30 @@ const RecruitCreatePage = () => {
 			isSubmitted: true,
 		}));
 
-		// const uploadPost = useMutation({
-		// 	mutationFn: () => postingRecruit(recruitFormData),
-		// });
-		// navigate(`/recruit/${uploadPost}`);
+		const postAvailable =
+			validCheck.isCategory &&
+			validCheck.isDeadline &&
+			validCheck.isProcedure &&
+			validCheck.isScope &&
+			validCheck.isTag &&
+			validCheck.isTitle;
+
+		if (postAvailable) {
+			uploadPost.mutate(formData as any);
+			navigate(`/recruit/${uploadPost}`);
+		}
```

Comment:
```suggestion
		if (postAvailable) {
			uploadPost.mutate(formData);
		}
```

로직을 바깥으로 빼어 함수의 역할이 더 잘보이도록 리팩토링해보았습니다!